configs/sst_kernel_rats-kernel-rt-base.yaml

```diff
@@ -243,3 +238,44 @@ data:
           - openssl-devel
         limit_arches:
           - x86_64
+
+      tuned-profiles-realtime:
+        description: This package is not in Fedora (yet), but we want it here.
+        requires:
+          - bash
+          - man
+        limit_arches:
+          - noarch
```

Comment:
`noarch` is not a valid architecture here. This field is not the "arch" part of the RPM name, it determines for what actual architectures is this package present.

Reply:
Ok, I'll drop all those noarch sections